src/RadixDlt.CoreApiSdk/core-api-spec-copy.yaml

```diff
@@ -73,7 +73,7 @@ info:
     [Gateway API](https://docs-babylon.radixdlt.com/).
 
     ## Integration and forward compatibility guarantees
-  
+
```

Comment:
i'm missing upgrade of deployment ci.env file with pointer to proper node version.

Reply:
Updated ci.env